### PR TITLE
added responsive cv

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Naufil CV</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
```
Comment (faran4engg) [inline index.html:14]: do u know what is this crossorigin and referrerpolicy ?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="left_side">
```
Comment (faran4engg) [inline index.html]: how about using aside tag ?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+        <div class="profileText">
+          <div class="imgBx">
+            <img src="face.svg" alt="Profile image" />
+          </div>
+          <h2>Naufil Shemle<br /><span>Web Developer</span></h2>
+        </div>
+
+        <div class="contactInfo">
```
Comment (faran4engg) [inline index.html]: section tag ?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+            <li>
+              <span class="text">Marathi</span>
+              <span class="percent">
+                <div style="width: 80%"></div>
+              </span>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="right_side">
```
Comment (faran4engg) [inline index.html]: same here

File: index.html [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+            <div class="text">
+              <p>
+                Making a E-commerce website while learing is one the best
+                practice to test your skills that's why I also tried this. Click
+                on the project name to visit actual porject.
+              </p>
+            </div>
+          </div>
+
+          <div class="box box1">
```
Comment (faran4engg) [inline index.html]: give meaningful name

File: style.css [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+body {
+  background-color: #c996cc;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
```
Comment (faran4engg) [inline style.css]: No grouping ?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+body {
+  background-color: #c996cc;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (faran4engg) [inline style.css:22]: why giving this to body ?

File: style.css [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+.container {
+  position: relative;
+  width: 100%;
+  max-width: 1000px;
+  min-height: 1000px;
+  background-color: #fff;
+  margin: 50px;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  box-shadow: 0 35px 55px rgba(0, 0, 0, 0.2);
```
Comment (faran4engg) [inline style.css:39]: no grouping ?

File: style.css [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+}
+
+.container .left_side {
+  position: relative;
+  background-color: #1c0c5b;
```
Comment (faran4engg) [inline style.css:44]: can you use css variables ?

File: style.css [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+
+.contactInfo ul li .icon {
+  display: inline-block;
+  width: 30px;
+  font-size: 18px;
+  color: #c996cc;
+}
+
+.contactInfo ul li span {
+  color: #c996cc;
```
Comment (faran4engg) [inline style.css]: css variables ?

File: style.css [hunk 11]
```diff
@@ -0,0 +1,269 @@
...
+  font-weight: 300;
+}
+
+.contactInfo.education li {
+  margin-bottom: 15px;
+}
+
+.contactInfo.education h5 {
+  color: #916bbf;
```
Comment (faran4engg) [inline style.css]: css variables ?

**Review by faran4engg — COMMENTED** [review]
@Naufil-Shemle  left some comments !

File: index.html [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+            <li>
+              <span class="icon"
+                ><i class="fa fa-map-marker" aria-hidden="true"></i
+              ></span>
+              <span class="text">Mumbai, India</span>
+            </li>
+          </ul>
+        </div>
+
+        <div class="contactInfo education">
```
Comment (syedabdullahali) [inline index.html]: Why u did not use semantic tags

File: index.html [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+            <h4>JavaScript</h4>
+            <div class="percent">
+              <div style="width: 50%"></div>
+            </div>
+          </div>
+
+          <div class="box">
+            <h4>ES6</h4>
+            <div class="percent">
+              <div style="width: 45%"></div>
```
Comment (murtazshaikh) [inline index.html]: Can you write all this styling in CSS ?

**Review by Farzebk — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (Naufil-Shemle) [reply index.html]: @faran4engg 
OK, done,

File: index.html (hunk 3 above)
Comment (Naufil-Shemle) [reply index.html]: @faran4engg
OK, done,

File: index.html (hunk 4 above)
Comment (Naufil-Shemle) [reply index.html]: @faran4engg
OK, done,

File: index.html (hunk 5 above)
Comment (Naufil-Shemle) [reply index.html]: @faran4engg
OK, done.

File: style.css (hunk 7 above)
Comment (Naufil-Shemle) [reply style.css:22]: @faran4engg For just this single page project I given to the body

File: style.css (hunk 6 above)
Comment (Naufil-Shemle) [reply style.css]: Now grouped.

File: style.css (hunk 8 above)
Comment (Naufil-Shemle) [reply style.css:39]: Now grouped.

File: style.css (hunk 9 above)
Comment (Naufil-Shemle) [reply style.css:44]: Yes, done.

File: style.css (hunk 10 above)
Comment (Naufil-Shemle) [reply style.css]: Yes, done.

File: style.css (hunk 11 above)
Comment (Naufil-Shemle) [reply style.css]: Yes, done.

File: style.css (hunk 7 above)
Comment (faran4engg) [reply style.css:22]: no, body should not have these css @Naufil-Shemle

File: index.html (hunk 1 above)
Comment (faran4engg) [reply index.html:14]: this ? @Naufil-Shemle

File: index.html [hunk 14]
```diff
@@ -0,0 +1,229 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="container">
```
Comment (faran4engg) [inline index.html]: main ?

Comment (Naufil-Shemle) [reply index.html]: Sorry I am not getting you point, could you please elaborate more ?

File: style.css (hunk 7 above)
Comment (Naufil-Shemle) [reply style.css:22]: OK Sir, updated.

File: index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply index.html:14]: @faran4engg
crossorigin is used to safely transfer the data between cross-domain.
referrerpolicy is used to set which information will be share by this website.

**Review by murtazshaikh — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

**Review by OSAMA478 — APPROVED** [review]
it looks great, the colors used could be more attractive but it differs from person to person.
overall it looks great to me.
excellent work naufil.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,231 @@
...
+            <div class="box project">
+              <div class="year_company">
+                <a
+                  href="https://naufil-shemle.github.io/Glowing-ring/"
+                  target="_self"
+                  ><h4>Loading ring</h4></a
+                >
+              </div>
+              <div class="text">
+                <h4></h4>
```
Comment (OSAMA478) [inline index.html]: The empty h4 tag is here.

File: index.html (hunk 13 above)
Comment (Naufil-Shemle) [reply index.html]: As we know that there are 3 types of CSS styling and inline css is used to specifically design a particular element that's why used this styling.

File: index.html (hunk 15 above)
Comment (Naufil-Shemle) [reply index.html]: Thanks to point out this.

**Review by ManSOorcode — APPROVED** [review]
you can link your linkedin and github account in your cv but overall presentation plus code of cv is good.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,230 @@
...
+                </span>
+              </li>
+              <li>
+                <span class="text">Marathi</span>
+                <span class="percent">
+                  <div style="width: 80%"></div>
+                </span>
+              </li>
+            </ul>
+          </section>
```
Comment (OSAMA478) [inline index.html:102]:
```suggestion
          <section class="contactInfo lang">
            <h3 class="title">Languages</h3>
            <ul>
              <li>
                <span class="text">Urdu</span>
                <div class="percent">
                  <div style="width: 90%"></div>
                </div>
              </li>
              <li>
                <span class="text">English</span>
                <div class="percent">
                  <div style="width: 80%"></div>
                </div>
              </li>
              <li>
                <span class="text">Hindi</span>
                <div class="percent">
                  <div style="width: 85%"></div>
                </div>
              </li>
              <li>
                <span class="text">Marathi</span>
                <div class="percent">
                  <div style="width: 80%"></div>
                </div>
              </li>
            </ul>
          </section>
```

I think you shouldn't nest a block-level element inside an inline element...

(block-level elements may contain inline or block) and
(inline may contain inline only)

File: index.html (hunk 14 above)
Comment (faran4engg) [reply index.html]: @Naufil-Shemle  ?

**Pull request merged**